### DISCUSSION TITLE
After Implementing JSF Template, Create Dialog Appears Disabled

### DISCUSSION THREAD

**Discussion by NCIDevRonR** [primefaces/community#251] [PrimeFaces] [answered]
Please reference [this reproducer](https://github.com/NCIDevRonR/primefaces-test/tree/cannot-select-row-on-data-table).  After successfully implementing a JSF template with a different reproducer, I've implemented it with this screen as well.  It all works fine until I click "Create" or "Update".
![Create Dialog Appears Disabled-20231016](https://github.com/primefaces/community/assets/78611742/f0f9e703-c76a-46b1-ae8f-e5c5da6fae09)
The modal dialog for creating or editing appears disabled, and the only new thing is the tags which associate the screen with the template file.  Is there something I can change in the dialog to make it appear enabled as it should?
An addendum:  I have updated [the reproducer](https://github.com/NCIDevRonR/primefaces-test/tree/cannot-select-row-on-data-table).  If I remove or set to false the Create/Edit dialog's `modal` property, the dialog appears enabled, meaning the user can enter and save a new row or a change to a row..  
![Non-Modal Create Dialog Appears Enabled-20231016](https://github.com/primefaces/community/assets/78611742/1afc0dfb-b985-4411-92ff-1e1db1d7cccf)
...which can wreak havoc in an application, because the main list screen is also enabled, and a different row can be selected while the dialog is open if it's not modal.  Again, if the row Create/Edit dialog is declared as modal, it appears disabled, but if it's non-modal, it appears enabled.  To me this seems like a defect, since the screen and the JSF template seem to be correctly implemented.
Has anyone seen this behavior?  Is there a known defect covering this?

**Comment by melloware** [comment, marked as answer]
@NCIDevRonR i fixed your demo:
[discussions-251.zip](https://github.com/user-attachments/files/16119323/discussions-251.zip)

Basically you were declaring dialogs INISDE the `pe:layout` which you don't want to do.  So I created the Dialogs outside of your layout now and they work fine with modal!
